modules/hfs/src/or_utils/or_vector.hpp

```diff
@@ -0,0 +1,92 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef _OPENCV_OR_VECTOR_HPP_
+#define _OPENCV_OR_VECTOR_HPP_
+
+#ifdef __CUDACC__
+#define __CV_CUDA_HOST_DEVICE__ __host__ __device__ __forceinline__
+#else
+#define __CV_CUDA_HOST_DEVICE__
+#endif
+
+namespace cv { namespace hfs { namespace orutils {
+
+
+template <class T>
+struct Vector2_ {
+    T x, y;
+};
+
+template <class T>
+struct Vector4_ {
+    T x, y, z, w;
+};
+
+template <class T> class Vector2 : public Vector2_ < T >
+{
+public:
+    __CV_CUDA_HOST_DEVICE__ Vector2() {}
+
+    __CV_CUDA_HOST_DEVICE__ Vector2(const T v0, const T v1)
+    {
+        this->x = v0;
+        this->y = v1;
+    }
+
+    __CV_CUDA_HOST_DEVICE__ Vector2(const Vector2_<T> &v)
+    {
+        this->x = v.x;
+        this->y = v.y;
+    }
+
+    __CV_CUDA_HOST_DEVICE__ friend Vector2<T> &operator /= (Vector2<T> &lhs, T d)
+    {
+        if (d == 0) return lhs; lhs.x /= d; lhs.y /= d; return lhs;
```

Comment:
Static code analyzers / modern compilers dislike this line (they warn about missing `{}`).
Please split this line on 2-3 lines.

```
modules/hfs/src/or_utils/or_vector.hpp: In function 'cv::hfs::orutils::Vector2<T>& cv::hfs::orutils::operator/=(cv::hfs::orutils::Vector2<T>&, T)':
modules/hfs/src/or_utils/or_vector.hpp:46:9: warning: this 'if' clause does not guard... [-Wmisleading-indentation]
         if (d == 0) return lhs; lhs.x /= d; lhs.y /= d; return lhs;
         ^~
```

Reply:
ok, I'll fix it right away :)